### PR TITLE
Fix to chart.ts & Ehr-view

### PR DESCRIPTION
**Pull request by mhigbyflowers**
Fixes the colors to the gender identity charts.
Improves chart.component functionality

### CODE REVIEW

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 1]
```diff
@@ -374,30 +374,31 @@ export class ChartComponent implements OnChanges {
     }
     let data = [];
     let cats = [];
+
+    // LOOP CREATES DYNAMIC CHART VARS
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/chart/chart.component.ts:378]: this loop was not functioning correctly....

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 2]
```diff
@@ -374,30 +374,31 @@ export class ChartComponent implements OnChanges {
...
     for (const a of results) {
       // For normal Gender Analysis , the stratum2 is the gender . For ppi it is stratum5;
       let color = null;
-      if (this.chartTitle === 'Biological Sex') {
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/chart/chart.component.ts:380]: the group if statements were wrapped with this

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 3]
```diff
@@ -12,271 +12,234 @@ <h2 class="primary-display">{{title}}</h2>
...
-                <div class="secondary-display info-text tbl-d">
-                  <button class="icon-btn">
-                    <clr-icon [ngClass]="r.expanded?'is-solid':''" shape="bar-chart" class="bar-chart-icon"></clr-icon>
-                  </button>
-                </div>
-              </div>
-            </div>
-            <div (click)="expandRow(standardConcepts.concat(items),r)" class="secondary-display info-text aka"
-              [ngClass]="{'no-b-border': r.expanded}" *ngIf="r.conceptSynonyms;else noConceptSynonyms">
-              <div *ngIf="synonymString[r.conceptId]" (click)="$event.stopPropagation();toggleSynonyms(r.conceptId)" class="aka-text"
```
Comment (mhigbyflowers) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:186]: IDK how this happened 🤷‍♂️ But it's fixed

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good. thanks

**Pull request merged**